common-npm-packages/packaging-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "azure-pipelines-tasks-packaging-common",
-    "version": "3.241.1",
+    "version": "3.242.1",
```

Comment:
3.242.0 ?

Reply:
The final digit represents the "Count of changes was made during the sprint" -- so that means it must always be > 0?
https://mseng.visualstudio.com/AzureDevOps/_wiki/wikis/AzureDevOps.wiki/29720/Mitigate-the-issue-Override-task-version

Reply:
my interpretation did not hold up to observed practice, so going with a zero here